### PR TITLE
Revert "[CI] Comment flaky tests (#10084)"

### LINKED ISSUE #10515: Remove `pytest-random-order` from conda tests
## 🚀 Tiny enhancement in CI
#10084 somehow fixed #10129 by randomise the order of the tests using `pytest-random-order`, but we should identify the source of the problem of conda CI runs getting killed.

### Pitch
- Identify the cause of #10129 
- Remove the change added in #10084
https://github.com/PyTorchLightning/pytorch-lightning/blob/fabb3644027a12c0df9acbdbc752d7ca7673396c/.github/workflows/ci_test-conda.yml#L33
https://github.com/PyTorchLightning/pytorch-lightning/blob/fabb3644027a12c0df9acbdbc752d7ca7673396c/.github/workflows/ci_test-conda.yml#L47

### Additional context

______________________________________________________________________

#### If you enjoy Lightning, check out our other projects! ⚡

- [**Metrics**](https://github.com/PyTorchLightning/metrics): Machine learning metrics for distributed, scalable PyTorch applications.

- [**Lite**](https://pytorch-lightning.readthedocs.io/en/latest/starter/lightning_lite.html): enables pure PyTorch users to scale their existing code on any kind of device while retaining full control over their own loops and optimization logic.

- [**Flash**](https://github.com/PyTorchLightning/lightning-flash): The fastest way to get a Lightning baseline! A collection of tasks for fast prototyping, baselining, fine-tuning, and solving problems with deep learning.

- [**Bolts**](https://github.com/PyTorchLightning/lightning-bolts): Pretrained SOTA Deep Learning models, callbacks, and more for research and production with PyTorch Lightning and PyTorch.

- [**Lightning Transformers**](https://github.com/PyTorchLightning/lightning-transformers): Flexible interface for high-performance research using SOTA Transformers leveraging Pytorch Lightning, Transformers, and Hydra.

### PR DESCRIPTION
**Pull request by akihironitta**
This reverts commit ed9802643c5485a0f07d8376009410ae76076cc4.

## What does this PR do?
- Removes unnecessary `standalone=True`. (without this flag, conda jobs used to frequently fail as tracked in #10129)
- Closes #10515.

### Does your PR introduce any breaking changes? If yes, please list them.
None

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [n/a] Did you make sure to **update the documentation** with your changes? (if necessary)
- [n/a] Did you write any **new necessary tests**? (not for typos and docs)
- [n/a] Did you verify new and **existing tests pass** locally with your changes?
- [x] Did you list all the **breaking changes** introduced by this pull request?
- [n/a] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @borda

### CODE REVIEW

**Comment by akihironitta** [comment]
will continue to investigate this in akihironitta/pytorch-lightning#2.

File: tests/profiler/test_profiler.py [hunk 1]
```diff
@@ -333,7 +333,6 @@ def test_pytorch_profiler_trainer_ddp(tmpdir, pytorch_profiler):
         assert any(f"{local_rank}-validation_step" in f for f in files)
 
 
-@RunIf(special=True)
```
Comment (carmocca) [inline tests/profiler/test_profiler.py:336]: Hey @akihironitta!

The random seed is already removed from master. You can still check if this `@RunIf` is still necessary though

**Comment by tchaton** [comment]
Hey @akihironitta,

Any updates on this one ?

**Comment by akihironitta** [comment]
Still WIP but to run CI jobs, I'll mark this PR as ready for review.

**Comment by akihironitta** [comment]
Could someone add this to an appropriate milestone?

File: tests/profiler/test_profiler.py (hunk 1 above)
Comment (akihironitta) [reply tests/profiler/test_profiler.py]: Removed the check, and there seems no problem :)

**Comment by awaelchli** [comment]
@akihironitta In order to know that it's no longer flaky, we need to rerun it several times

**Comment by akihironitta** [comment]
@awaelchli I'll rerun the conda jobs (which were previously failing) three times now. Does that sound reasonable?

**Review by awaelchli — APPROVED** [review]

**Review by carmocca — APPROVED** [review]

**Review by Borda — APPROVED** [review]
